config/cluster.yaml.template

```diff
@@ -0,0 +1,11 @@
+# base cluster settings
+base:
+    cluster_id: <id>
```

Comment:
all of this is in the cluster file not sure if they all need to be prefixed with cluster. also the base key is probably not needed